timelines/games201/mass_spring_explicit.yml

```diff
@@ -0,0 +1,524 @@
+---
+- path: /Users/lance/python/taichi-release-tests/repos/games201/mass_spring_explicit.py
+  args: []
+  steps:
+  - {frame: 1, action: move, position: [0.561, 0.812]}
+  - {frame: 2, action: move, position: [0.561, 0.846]}
+  - {frame: 1, action: move, position: [0.564, 0.877]}
+  - {frame: 1, action: move, position: [0.572, 0.893]}
+  - {frame: 1, action: move, position: [0.58, 0.908]}
+  - {frame: 1, action: move, position: [0.592, 0.92]}
+  - {frame: 1, action: move, position: [0.598, 0.926]}
```

Comment:
Please cut redundant moves and actions as much as possible, to reduce running time.

Reply:
got it, thx

Reply:
done